### DISCUSSION TITLE
New config option to allow teleporting users to spawn when their jail time expires

### DISCUSSION THREAD

**Discussion by baailey** [EssentialsX/Essentials#4123] [Ideas and Feature Suggestions] [answered]
Currently, the only config option related to a user being TP'd out of jail is `teleport-back-when-freed-from-jail`. This option when set to **true** will teleport a user back to where they were before a jail, which I personally find inconvenient for situations where a user is jailed for xray. It'll simply teleport them back to the mine they were jailed for. 

If you set this option to **false**, the user will remain in jail until they themselves teleport out with /spawn, /home etc (whatever the server allows them access to), which is another inconvenience if the user does not know this is an option. Can we set up signs / holos etc to let a user know about this? Yes. However, that could also not fit the theme of certain jails. 

So, I propose a new config option (or changing the behaviour of the **false** toggle for `teleport-back-when-freed-from-jail`) that will automatically send users to spawn when their jail time expires.

**Comment by JRoy** [comment, marked as answer]
Will be added in #4132
